config/locales/miners/miner.en.yml

```diff
@@ -66,3 +66,9 @@ en:
         open: Open
         closed: Closed
         merged: Merged
+    vacation:
+      status:
+        pending: Pending
+        approved: Approved
+        refused: Refused
+        canceled: Canceled
```

Comment:
in this translation file we're calling it `canceled` with a single L, everywhere else we're calling `cancelled` with two L's, maybe change it to `cancelled` here as well?